src/main/java/io/codelex/TestAdvanced/Exercise4/Exercise4.java

```diff
@@ -0,0 +1,26 @@
+package io.codelex.TestAdvanced.Exercise4;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+
+public class Exercise4 {
```

Comment:
Nu šis uzdevums gan īsti nedara to, kas bija prasīts - prasīts bija nolasīt vienu failu un visu saturu apgriezt otrādi (tai skaitā faila nosaukumu) un to ierakstīt otrā failā.

Reply:
Hey, salaboju.